pulsar-broker/src/main/java/org/apache/pulsar/broker/resourcegroup/ResourceGroupService.java

```diff
@@ -539,29 +549,32 @@ private void initialize() {
                     this.timeUnitScale);
         this.maxIntervalForSuppressingReportsMSecs =
                 this.resourceUsagePublishPeriodInSeconds * this.MaxUsageReportSuppressRounds;
+
     }
 
-    private void checkRGCreateParams(ResourceGroupConfigInfo rgConfig) throws PulsarAdminException {
+    private void checkRGCreateParams(String rgName, org.apache.pulsar.common.policies.data.ResourceGroup rgConfig)
+      throws PulsarAdminException {
         try {
             checkNotNull(rgConfig);
         } catch (NullPointerException e) {
             throw new IllegalArgumentException("ResourceGroupCreate: Invalid null ResourceGroupConfigInfo");
         }
 
-        if (rgConfig.getName().isEmpty()) {
+        if (rgName.isEmpty()) {
             throw new IllegalArgumentException("ResourceGroupCreate: can't create resource group with an empty name");
         }
 
-        ResourceGroup rg = getResourceGroupInternal(rgConfig.getName());
+        ResourceGroup rg = getResourceGroupInternal(rgName);
         if (rg != null) {
-            throw new PulsarAdminException("Resource group already exists:" + rgConfig.getName());
+            throw new PulsarAdminException("Resource group already exists:" + rgName);
         }
     }
 
     private static final Logger log = LoggerFactory.getLogger(ResourceGroupService.class);
     private final PulsarService pulsar;
     protected final ResourceQuotaCalculator quotaCalculator;
-    private ResourceUsageTransportManager resourceUsageTransportMgr;
+    private ResourceUsageTransportManager resourceUsageTransportManagerMgr;
```

Comment:
Nit: didn't follow why there is a 2-level manage in the name (`ManageMgr`).

Reply:
This name was generated by the IDE (probably to differentiate the objects of the classes for which there is a name clash). I assume there must be some convention that IDE is picking and let the name be what was generated instead of changing it.

Reply:
My vote would be to give it a meaningful name; "ManagerMgr" sounds odd, even if it is picked by an IDE.